share/spice/drupal/drupal.js

```diff
@@ -0,0 +1,78 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_drupal = function(api_result){
+
+        // Validate the response
+        if (!api_result || api_result.list === "") {
+            return Spice.failed('drupal');
+        }
+
+        // Render the response
+        Spice.add({
+            id: "drupal",
```

Comment:
Unfortunately we already have an ID called "drupal". Since we don't have namespacing set up to prevent this, would it be ok if you changed the ID to something like "drupal_search"? Same goes for the argument in `Spice.failed`. Thanks!